### ISSUE TITLE
Sources

### ISSUE THREAD

**Issue by TechStudent11** [KendallDoesCoding/languages-around-the-world#4]
You should cite your sources for where you got your info

**Comment by KendallDoesCoding** [comment]
> You should cite your sources for where you got your info

Good idea, but I mostly get my sources from "People May Ask" on google, and those pages have other info too, so if I go through every site and cite the info it may take a little extra time, but something to consider.

**Comment by TechStudent11** [comment]
`possibly-feature`

You should still cite your sources, because what you are doing right now is plagiarism.

**Comment by KendallDoesCoding** [comment]
> `possibly-feature`
> 
> You should still cite your sources, because what you are doing right now is plagiarism.

Yup true. I did pretty much the same as this is rn in my Orange Wiki and that stopped work on the project, I don't plan on doing too much work on this project too though so yeah lol.

**Comment by KendallDoesCoding** [comment]
Added the on-hold label, but trust me, I'm working on this and will come out before the next language added, if I add a new language.
